.gitignore

```diff
@@ -0,0 +1,4 @@
+
+target/
```

Comment:
tbh I thought we would need a different pattern, but this worked on my local repo

Reply:
if it works, it works